docs/ampform/LambdaKpi0.ipynb

```diff
@@ -99,101 +95,138 @@
+    "latex_particle_names = {\n",
+    "    \"Lambda\": r\"\\Lambda\",\n",
+    "    \"K+\": r\"K^+\",\n",
+    "    \"pi0\": r\"\\pi^0\",\n",
+    "    \"gamma\": r\"\\gamma\",\n",
+    "    \"p\": \"p\",\n",
+    "}\n",
+    "\n",
+    "\n",
+    "def format_parity(parity: int | None) -> str:\n",
+    "    if parity is None:\n",
+    "        return \"\"\n",
+    "    if parity == -1:\n",
+    "        return \"-\"\n",
+    "    if parity == 1:\n",
+    "        return \"+\"\n",
+    "    return str(parity)\n",
+    "\n",
+    "\n",
+    "def format_fraction(value: Fraction | None) -> str | None:\n",
+    "    if value is None:\n",
+    "        return None\n",
+    "    if value.denominator == 1:\n",
+    "        return str(value.numerator)\n",
+    "    return rf\"\\frac{{{value.numerator}}}{{{value.denominator}}}\"\n",
+    "\n",
+    "\n",
+    "def format_fraction_isospin3(value: Fraction | None) -> str | None:\n",
+    "    if value is None:\n",
+    "        return None\n",
+    "    if value.denominator == 1:\n",
+    "        return str(value.numerator)\n",
+    "    return rf\"$\\frac{{{value.numerator}}}{{{value.denominator}}}$\"\n",
+    "\n",
+    "\n",
+    "def particle_extract(name: str) -> dict:\n",
+    "    particle = particle_db[name]\n",
+    "\n",
+    "    isospin_magnitude_value = (\n",
+    "        Fraction(particle.isospin.magnitude) if particle.isospin is not None else None\n",
+    "    )\n",
+    "    isospin_magnitude_latex = (\n",
+    "        format_fraction(isospin_magnitude_value)\n",
+    "        if isospin_magnitude_value is not None\n",
+    "        else \"0\"\n",
+    "        if particle.isospin and particle.isospin.magnitude == 0\n",
+    "        else \"\"\n",
+    "    )\n",
+    "\n",
+    "    isospin_projection_value = (\n",
+    "        Fraction(particle.isospin.projection) if particle.isospin is not None else None\n",
+    "    )\n",
+    "    isospin_projection_latex = (\n",
+    "        format_fraction_isospin3(isospin_projection_value)\n",
+    "        if isospin_projection_value is not None\n",
+    "        else \"0\"\n",
+    "        if particle.isospin and particle.isospin.projection == 0\n",
+    "        else None\n",
+    "    )\n",
+    "\n",
+    "    spin_fraction_value = Fraction(particle.spin) if particle.spin is not None else None\n",
+    "    spin_latex = (\n",
+    "        format_fraction(spin_fraction_value)\n",
+    "        if spin_fraction_value is not None\n",
+    "        else \"0\"\n",
+    "        if particle.spin == 0\n",
+    "        else \"\"\n",
+    "    )\n",
+    "\n",
+    "    if particle.isospin is not None:\n",
+    "        jpc_ig = rf\"${spin_latex}^{{{format_parity(particle.parity)}{format_parity(particle.c_parity)}}} \\ ({isospin_magnitude_latex}^{{{format_parity(particle.g_parity)}}})$\"\n",
+    "    else:\n",
+    "        jpc_ig = rf\"${spin_latex}^{{{format_parity(particle.parity)}{format_parity(particle.c_parity)}}}$\"\n",
```

Comment:
I would put all this stuff in a function that does `Particle` -> `str` $J^PC(I^G)$